docs/custom-setups/private_net_tutorial.rst

```diff
@@ -65,21 +65,21 @@ Prepare a file ``genesis.json`` with the following content.
  <snip>/priv_chain
  (env) $ cat genesis.json
  {
-"config": {
-"chainId": 4321,
-"homesteadBlock": 0,
-"eip150Block": 0,
-"eip155Block": 0,
-"eip158Block": 0,
-"ByzantiumBlock": 0,
-"eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
-"ConstantinopleBlock": 0,
-"PetersburgBlock": 0
-},
-"alloc": {},
-"difficulty" : "0x1",
-"gasLimit"   : "0x9880000"
-}
+ "config": {
```

Comment:
This formatting is weird, why add spaces there?

Reply:
If we dont add spaces it will not be properly formatted

Reply:
But then do it consistenly, after `"config": {` there is a new level of indentation. and usually indention is at least 2 spaces.

Reply:
Ok I just saw it. I have given consistent spacing after `config`. I also had a look at this file as a whole. On this file go to the top right corner 3 dots. Click on view file. It is showing up correctly. Rest I also trying to run this locally and see if documentation comes out correctly but atleast in github it is turning out correctly.

Reply:
![Screen Shot 2021-08-10 at 14 22 55](https://user-images.githubusercontent.com/90851/128866087-91858f89-2ea8-4db8-8fb3-c70d4e569fdd.png)
This is what I see in the diff, and this is **not** correctly indented.